gtk/gtk_export.go

```diff
@@ -1,14 +1,42 @@
 package gtk
 
-// #cgo pkg-config: gtk+-3.0
-// #include <gtk/gtk.h>
+/*
+ #cgo pkg-config: gtk+-3.0
+ #include <gtk/gtk.h>
+*/
 import "C"
 import (
+	"strings"
 	"unsafe"
 
 	"github.com/gotk3/gotk3/glib"
 )
 
+//export goStringMatch
+func goStringMatch(model *C.GtkTreeModel,
+	column C.gint,
+	key *C.gchar,
+	iter *C.GtkTreeIter,
+	data C.gpointer) C.gboolean {
+
+	goModel := &TreeModel{glib.Take(unsafe.Pointer(model))}
```

Comment:
`glib.Take` will increase the reference count, but it will never be decreased. You need to either `defer goModel.Unref()` or remove the `Take`.